### PR TITLE
Adjust the details of the dora writing process for extension

### PR DESCRIPTION
**Pull request by Jackson-Wang-7**
### What changes are proposed in this pull request?

Please outline the changes and how this PR fixes the issue.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

**Comment by yyongycy** [comment]
looks good overall!

**Review by yyongycy — APPROVED** [review]
lgtm

File: dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java [hunk 1]
```diff
@@ -736,7 +738,8 @@ private void invalidateFileMeta(String path) {
   }
 
   @Override
-  public void completeFile(String path, CompleteFilePOptions options, String uuid) {
+  public void completeFile(String path, CompleteFilePOptions options, String uuid)
+      throws IOException, AccessControlException {
```
Comment (yyongycy) [inline dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java:742]: how about the rest operations? Add exception next time?

Comment (Jackson-Wang-7) [reply dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java:742]: maybe next time, I can see there will be more APIs added.

**Comment by Jackson-Wang-7** [comment]
alluxio-bot, merge this please

**Pull request merged**